src/ucs/vfs/base/vfs_obj.c

```diff
@@ -760,6 +760,16 @@ ucs_status_t ucs_vfs_path_get_link(const char *path, ucs_string_buffer_t *strb)
     return status;
 }
 
+static void ucs_vfs_obj_init_globals()
+{
+    ucs_global_opts_vfs_init();
```

Comment:
maybe just add to the UCS_STATIC_INIT of global_opts.c?
then, vfs will have to init itself in the first time it's being used, instead of by calling ucs_vfs_node_init explicitly

Reply:
If vfs is initialized along with the first use, then clean up should be done in `ucs_vfs_obj_remove`. And checking for emptiness should be added into the method. Is this ok?

Reply:
maybe global_opts will not bother to remove anything itself, just vfs will remove any remaining nodes when it cleans up?